### PR TITLE
moving all developer-focused docs into EMOD code repo

### PR DESCRIPTION
**Pull request by JSchripsema-IDM**
This is currently building without error, but only contains links back to the generic EMOD docs. Links to the HIV and malaria docs are in comments until they can be merged into emodpy-hiv and emodpy-malaria. Once those are merged, I can update those commented links and this PR can be reviewed.

### CODE REVIEW

**Review by Bridenbecker — CHANGES_REQUESTED** [review]
There are just a couple of small things to clean up.  Thank you for saving these docs.  I did not know that some of it was even there.

File: docs/conf.py
Comment (JSchripsema-IDM) [reply docs/conf.py]: I just missed it, removed now.

File: docs/_templates/footer_start.html
Comment (JSchripsema-IDM) [reply docs/_templates/footer_start.html:2]: Fixed this, docset name, and license.

File: README.md
Comment (JSchripsema-IDM) [reply README.md]: Just a mistake on my part!

**Comment by JSchripsema-IDM** [comment]
@Bridenbecker Thanks Dan for the thorough review! I addressed all of your comments. Once all three related EMOD PRs are merged, I'll update all the links that were commented out.

**Comment by Bridenbecker** [comment]
@JSchripsema-IDM  FYI.  I'm assuming that **you** will merge when you think it is ready.

**Pull request merged**
